### ISSUE TITLE
Importing separate tll file in Netedit doesn't work, with SUMO or SUMO-GUI does work

### ISSUE THREAD

**Issue by ZoltanBaksa** [eclipse/sumo#4622]
In our Stuttgart-simulation project we have a basic map with netedit defined TLS programs. In the map the tls controlled junctions are not joined, only tls joined, where the TL attribute is set accordingly. There is a separate tll additional file where some important tls joins are having their own program sets and wauts.  When the separate tll additional file is set in the sumo configuration file, it is working perfectly in sumo and sumo-gui, but netedit is not able to use it, because it cannot match the junctions to the waut-juncitons.

**Comment by namdre** [comment]
so to rephrase this: Your issue is that netedit does not handle WAUT. correct?

**Comment by ZoltanBaksa** [comment]
Not exactly. The matching of waut's junctionID attribute works in sumo and sumo-gui, but not at all in netedit. The TL name information is stored in the connection definitions as reference to tlLogic tag's id attribute, not on the level of junction definition where only the type is set to 'traffic_light'. It would be more accurate, if the wautJunction tag would reference the tlLogic tag instead of the junction

**Comment by namdre** [comment]
Despite the attribute name 'junctionID' the attribute actually references a tlLogic id so the implementation works as intended, only the attribute name is crap.

Netedit doesn't know anything about WAUT and doesn't even try to parse them.
However, loading the signal plans themselves should work.  (`Menu File->Traffic Lights->Load TLS Programs`)

**Comment by ZoltanBaksa** [comment]
That is what does not work, because netedit drops an error entry: 'ERROR, Junction doesn't exist', and the programs in traffic light editor remain the original.

**Comment by namdre** [comment]
please send the input files directly to me so I can take a look.

**Comment by namdre** [comment]
The initial error message does not originate with the  WAUT. Rather, netedit cannot load joined traffic lights via menu.
As a workaround, you can load the definitions via
 netedit -s net.net.xml -i tll.add.xml

Then the programs are loaded and work (as long as they have the correct state length).
The different programs can be selected in the tls editor frame and edited as required.
When saving the network, the additional definitions are embedded in the .net.xml file

When saving the tls definitions, all of them are written to a file but the WAUT are lost.
A possible solution would be to keep all the WAUT stuff in a seperate file.

**Comment by ZoltanBaksa** [comment]
Thanks, till there is a bugfix I'll do so.

**Comment by namdre** [comment]
@ZoltanBaksa  Have you tried this with the latest development version?

**Comment by ZoltanBaksa** [comment]
No, I used SUMO version: 1.0.1 + 0019-83bdcaf4ad
